oe_content.module

```diff
@@ -13,12 +13,12 @@ use Drupal\Core\Field\FieldStorageDefinitionInterface;
 use Drupal\link\LinkItemInterface;
 
 /**
- * Implements hook_entity_base_field_info().
+ * Implements hook_entity_bundle_field_info().
  */
-function oe_content_entity_base_field_info(EntityTypeInterface $entity_type) {
+function oe_content_entity_bundle_field_info(EntityTypeInterface $entity_type, $bundle, array $base_field_definitions) {
   $fields = [];
-
-  if ($entity_type->id() !== 'node') {
+  // Not alter non-coporate node bundles.
+  if ($entity_type->id() !== 'node' || strpos($bundle, 'oe_') !== 0) {
```

Comment:
This means that, if one site decides to call its bundle `oe_something` then it will get these base fields. We should move this hook into each content type's sub-module and properly match that content type bundle's ID.

Reply:
@ademarco 
Please take a look in my second suggestion.